packages/plugin-lighthouse/src/lib/runner/utils.unit.test.ts

```diff
@@ -145,13 +178,61 @@ describe('toAuditOutputs', () => {
       types.map(
         type =>
           ({
-            details: { type },
-          } as Result),
+            id: 'cumulative-layout-shift',
+            details: { type, headings: [], items: [] },
+            score: 0,
+            numericValue: 0,
+            displayValue: '0 ms',
+          } as unknown as Result),
```

Comment:
There's always a better way than `as unknown as T` (part 3):

```suggestion
            id: 'cumulative-layout-shift',
            details: { type, headings: [], items: [] } as Details,
            score: 0,
            numericValue: 0,
            displayValue: '0 ms',
          } as Result),
```